### PR TITLE
Mr-65-66-67  visual bug fixes

### PR DESCRIPTION
**Pull request by JaqKent**
visual fixes and updates

### CODE REVIEW

File: src/components/OnSaleSection/index.tsx [hunk 1]
```diff
@@ -25,7 +24,14 @@ function OnSaleSection() {
...
     return (
         <div className={styles.container}>
-            <p className={styles.onSale}>Oferta Del Dia</p>
+            <p className={styles.onSale}>
+                Ofertas
+                <img
+                    className={styles.icon}
+                    src={sale}
+                    alt="ofertas-icon"
+                />{' '}
```
Comment (nathan-hyan) [inline src/components/OnSaleSection/index.tsx:33]: Is the extra space necessary?

**Review by nathan-hyan — COMMENTED** [review]
Great job, some comments!

File: src/components/OnSaleSection/index.tsx (hunk 1 above)
Comment (JaqKent) [reply src/components/OnSaleSection/index.tsx:33]: done

File: src/components/OnSaleSection/index.tsx [hunk 2]
```diff
@@ -46,14 +52,7 @@ function OnSaleSection() {
...
             >
-                {viewMore ? (
-                    <span>Ir a Ofertas</span>
-                ) : (
-                    <span>
-                        {' '}
-                        Mostar Mas <FontAwesomeIcon icon={faChevronDown} />
-                    </span>
-                )}
+                {viewMore ? <span>Ir a Ofertas</span> : <span>Ver Mas</span>}
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/OnSaleSection/index.tsx:55]: done

**Pull request merged**